arrays/fizz_buzz_array.rb

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env ruby
+
+# Implement Fizz Buzz into an array in fizz_buzz_array.rb
+
+# output integers 1 through 100:
+# replacing FizzBuzz for numbers divisible by 3 and 5
+# replacing Fizz for numbers divisible by 3 only
+# replacing Buzz for numbers divisible by 5 only
+
+# instead of an iterator, use an array 1-100
+numbers = (1..100).to_a
+# value = 101
+
+# place output to an array
+fizz_buzz_output = []
+
+numbers.each do |value|
```

Comment:
Can you use map instead?

    fizz_buzz_output = numbers.map do |value|
      ...
    end
    puts fizz_buzz_output